src/test/resources/karate-config.js

```diff
@@ -1,12 +1,25 @@
 function config() {
+
     var env = karate.env; // get system property 'karate.env'
     karate.log('karate.env system property was:', env);
     karate.configure('ssl',true);
+
+    DB_URL: DB_URL,
+    DB_USERNAME:DB_USERNAME,
+    DB_PASSWORD:DB_PASSWORD,
+    DB_DRIVER:DB_DRIVER,
+    URL_PATH:URL_PATH
+
     if (!env) {
       env = 'dev';
     }
-    var data = read('classpath:karate/support/env_data.json');
+    var data = read('classpath:config');
     var env_data = data[env];
+    var  DB_URL = data.DB_URL;
+    var  DB_USERNAME = data.DB_USERNAME;
+    var  DB_PASSWORD = data.DB_PASSWORD;
+    var  DB_DRIVER = data.DB_DRIVER;
+    var  URL_PATH = data.URL_PATH;
 
     return data[env];
 }
```

Comment:
I still think that we should update this karate-config to make it more clear and usable:
1. how are those firsts `DB_URL: DB_URL, ...` supposed to work here? you are then declaring those again below, does it work if you run it?
2. is this reading a `config` file on the classpath? i think there isn't any file like that here
3. we are declaring then some variables `DB_URL`, `URL_PATH`, etc, assigning them some values based on the read file but not returning them to become 'global vars' for the features, what is the purpose for that?
4. this is also declaring env_data and not using it anymore then
5. the first karate.log will probably print a null value as you are probably expecting it to be empty and assigning it the value `dev` then, so its probably better to just swap their positions and log something more clear
6. reading a local file probably only works on local/dev so we should probably make different configurations for it if this is running on pipeline or similar